src/components/ArtifactIdentifierParser/ArtifactIdentifierParser.tsx

```diff
@@ -0,0 +1,62 @@
+import { Flex, FlexItem, Label } from '@patternfly/react-core';
+
+import { Artifact } from 'pnc-api-types-ts';
+
+interface IArtifactIdentifierParserProps {
+  artifact: Artifact;
+}
+
+export const ArtifactIdentifierParser = ({ artifact }: IArtifactIdentifierParserProps) => {
+  const artifactNameSplitted = artifact.identifier.split(':');
+  const repoType = artifact.targetRepository?.repositoryType;
+
+  if (repoType === 'MAVEN') {
+    return (
+      <Flex spaceItems={{ default: 'spaceItemsXs' }}>
```

Comment:
use separate `const` variable with the proper type, the same for all other cases